mxcubeweb/core/components/queue.py

```diff
@@ -1582,7 +1582,9 @@ def add_queue_entry(self, node_id, task, task_name):
             )
 
         full_path, process_path = HWR.beamline.session.get_full_path(
-            os.path.join(params.get("subdir", ""), params.get("experiment_name", "")),
+            os.path.join(
+                params.get("subdir", ""), params.get("experiment_name", "") or ""
```

Comment:
add a comment here, that explains that `experiment_name` can either be omitted or be `None`